core/geometry/src/curve/internalContexts/geometry2d/ImplicitCurve2dLocationDetail.ts

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module CartesianGeometry
+ */
+
+
+import { Angle } from "../../../geometry3d/Angle";
+import { Point2d, Vector2d } from "../../../geometry3d/Point2dVector2d";
+import { ImplicitCurve2d } from "./implicitCurve2d";
+
+/**
+ * ImplicitCurve2dLocationDetail carries a curve reference and further details about 
+ * a point on the curve.
+ * The "further information" optionallly includes a Point2d, an Angle, and an additional Vector2d, 
+ */
+export class ImplicitCurve2dLocationDetail {
```

Comment:
I'm trying to understand class data:

`_curve` is the curve we are trying to find the tangent to?
`_angle` is the angle between curve and tangent line?
-_point` is where the tangent hits the curve?
`_tangentVector` is the tangent?
- what are `_gradientVector` and `_functionValue`?

would you please add some docs?

Reply:
THis is an analogy of the familiar PointLocationDetail for Curve3d.   The intention is to use it to indicate positions along one of the 2D curves.  I'm hoping to implement that in a refactoring roud.

Reply:
(docs added)